.github/workflows/publish.yml

```diff
@@ -10,16 +10,16 @@ permissions:
 
 jobs:
   publish_maplibre_gl_platform_interface:
-    uses: ./.github/workflows/publish_single.yml
+    uses: .github/workflows/publish-single.yml@main
     with:
       working-directory: maplibre_gl_platform_interface
   publish_maplibre_gl_web:
     needs: [publish_maplibre_gl_platform_interface]
-    uses: ./.github/workflows/publish_single.yml
+    uses: .github/workflows/publish-single.yml@main
     with:
       working-directory: maplibre_gl_web
   publish_maplibre_gl:
     needs: [publish_maplibre_gl_platform_interface, publish_maplibre_gl_web]
-    uses: ./.github/workflows/publish_single.yml
+    uses: .github/workflows/publish-single.yml@main
```

Comment:
```suggestion
    uses: ./.github/workflows/publish-single.yml
```

Reply:
fixed, used your solution in ceb8ba86d2067fe97628f5f25ca2f948c03fbb4a